package.json

```diff
@@ -109,6 +111,7 @@
     "null-loader": "3.0.0",
     "optimize-css-assets-webpack-plugin": "5.0.3",
     "postcss-loader": "3.0.0",
+    "prettier": "^1.18.2",
```

Comment:
can you please fix the version to 1.18.2 (Remove the `^`) the same for the others two dependencies. Thanks

Reply:
ok ill remove it